generators/client/files-angular.js

```diff
@@ -315,6 +315,9 @@ const files = {
                 'shared/alert/alert-error.model.ts',
                 // dates
                 'core/date/datepicker-adapter.ts',
+                // base
+                'shared/base.component.ts',
+                'shared/base.directive.ts',
```

Comment:
No need to create multiple classes for unsubscription. It's enough to create one base class like SubscriptionDestroyer.